crates/editor/src/editor_tests.rs

```diff
@@ -4519,6 +4519,120 @@ async fn test_autoclose_pairs(cx: &mut gpui::TestAppContext) {
     cx.assert_editor_state("a\"\"ˇ");
 }
 
+#[gpui::test]
+async fn test_always_handle_autoclosed_character_skip_over(cx: &mut gpui::TestAppContext) {
+    init_test(cx, |settings| {
+        settings.defaults.always_handle_autoclosed_character = Some(true);
+    });
+
+    let mut cx = EditorTestContext::new(cx).await;
+    cx.update_editor(|view, _cx| view.set_always_handle_autoclosed_character(true));
+
+    let language = Arc::new(Language::new(
+        LanguageConfig {
+            brackets: BracketPairConfig {
+                pairs: vec![
+                    BracketPair {
+                        start: "{".to_string(),
+                        end: "}".to_string(),
+                        close: true,
+                        newline: true,
+                    },
+                    BracketPair {
+                        start: "(".to_string(),
+                        end: ")".to_string(),
+                        close: true,
+                        newline: true,
+                    },
+                    BracketPair {
+                        start: "/*".to_string(),
+                        end: " */".to_string(),
+                        close: true,
+                        newline: true,
+                    },
+                    BracketPair {
+                        start: "[".to_string(),
+                        end: "]".to_string(),
+                        close: false,
+                        newline: true,
+                    },
+                    BracketPair {
+                        start: "\"".to_string(),
+                        end: "\"".to_string(),
+                        close: true,
+                        newline: false,
+                    },
```

Comment:
Do you need these here? Seems like you don't use them in the test.

Reply:
Probably we don't, copied from the the above